charts/intel-gpu-resource-driver/templates/resource-driver.yaml

```diff
@@ -70,11 +48,12 @@ spec:
           mountPath: /etc/cdi
         - name: varruncdi
           mountPath: /var/run/cdi
-        - name: defaults
-          mountPath: "/defaults"
-          readOnly: true
+        # when using fake sysfs - mount at the same place as on host
         - name: sysfs
           mountPath: "/sysfs"
+        # Only use devdri when using fake devfs with device-faker
+        #- name: devdri
+        #  mountPath: "/tmp/test-0123456789/dev/dri"
         securityContext:
```

Comment:
This does not belong in Helm chart templates, it is only used when testing / deploying DRA driver on nodes without hardware.

Reply:
This was for lines 54-57, not the securityContext